adapters.json

```diff
@@ -34,5 +34,10 @@
       "appnexus": {
         "alias": "pagescience"
       }
+    },
+    {
+      "appnexusAst": {
+        "supportedMediaTypes": ["video"]
+      }
```

Comment:
It seems an adapter would know it's supported types, unless that would be an optional config for a publisher to choose. The `adaptermanager._bidderRegistry` could be queried by a function exported by `adaptermanager` to get the array of `videoAdapters`.